mypy/build.py

```diff
@@ -1098,6 +1108,13 @@ def validate_meta(meta: Optional[CacheMeta], id: str, path: Optional[str],
     if data_mtime != meta.data_mtime:
         manager.log('Metadata abandoned for {}: data cache is modified'.format(id))
         return None
+    deps_mtime = None
+    if manager.options.cache_fine_grained:
+        assert meta.deps_json
+        deps_mtime = getmtime(meta.deps_json)
```

Comment:
Use the file system cache?

Reply:
We don't use the cache for meta files at all, currently